idc_index/index.py

```diff
@@ -1628,6 +1715,44 @@ def download_from_selection(
             list_of_directories=list_of_directories,
         )
 
+    def download_dicom_instance(
```

Comment:
Do we need this function? I think it might be good to deprecate those shell functions, and just keep `download_from_selection` - what do you think?

Reply:
I just implemented it in the first case as analogue to `download_dicom_series` etc. They are used in the tests currently, maybe that's the reason they were implemented in the first case?